tests/Config/GlobalQuery.test.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { GlobalQuery } from '../../src/Config/GlobalQuery';
+
+describe('GlobalQuery tests', () => {
+    it('getInstance() should return the same object', () => {
+        const globalQuery1 = GlobalQuery.getInstance();
+        const globalQuery2 = GlobalQuery.getInstance();
+
+        expect(Object.is(globalQuery1, globalQuery2)).toEqual(true);
+    });
+
+    it('should have an empty source by default', () => {
+        expect(new GlobalQuery().isEmpty()).toEqual(true);
```

Comment:
I would suggest also getting the query and confirming it is an empty string, just for safety.

Reply:
oh right! done =)